controllers/access_control.go

```diff
@@ -375,7 +384,7 @@ func (roleCreate AerospikeRoleCreateUpdate) createRole(client *as.Client, adminP
 		return fmt.Errorf("could not create role %s: %v", roleCreate.name, err)
 	}
 
-	err = client.CreateRole(adminPolicy, roleCreate.name, aerospikePrivileges, roleCreate.whitelist)
+	err = client.CreateRole(adminPolicy, roleCreate.name, aerospikePrivileges, roleCreate.whitelist, roleCreate.writeQuota, roleCreate.writeQuota)
```

Comment:
Using only roleCreate.writeQuota, should it be roleCreate.readQuota in one place?

Reply:
yes